css/styles.css

```diff
@@ -55,39 +60,307 @@ a {
 	clear: both;
 }
 
+/* Photodiary */
+
+.photo-diary {
+	width: 85%;
+	padding-bottom: 100px;
+	margin: 0 auto;
+	color: rgba(115, 115, 115, .9);
+}
+
+.photo-diary span {
+	display: block;
+	font-size: 0.9rem;
+	font-family: 'Ubuntu', sans-serif;
+	text-transform: uppercase;
+	opacity: .6;
+}
+
+.photo-diary h1 {
+	padding-bottom: 10px;
+	font-family: "Playfair Display", serif;
+	font-weight: normal;
+}
+
+.photo-diary p {
+	display: block;
+	padding-bottom: 40px;
+	font-family: "Playfair Display", serif;
+	font-size: 0.9rem;
+}
+
+
+/* Div with 4 articles */
+
+.articles-div {
+	width: 85%;
+	padding-top: 80px;
+	padding-bottom: 40px;
+	margin: 0 auto;
+	color: rgba(115, 115, 115, .9);
+}

```

Comment:
Sprowadziłem to do jednego elementu `.narrow` i ustawiłem dla niego formatowanie dla wszystkich media queries. 

Zmieniłem też dopiskę "div" na "section" więc teraz jest np. `articles-section`